net-libs/aqbanking/aqbanking-6.5.4.recipe

```diff
@@ -0,0 +1,123 @@
+SUMMARY="A library for online banking and financial applications"
+DESCRIPTION="AQBanking"
+HOMEPAGE="http://www.aqbanking.de"
+COPYRIGHT="Martin Preuss"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/aqbanking/aqbanking/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="66b13d7694c25d09511b77edc14262025a452dfd93359c7257826f0e4c60dc6b"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+libVersion="44.5.4"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	aqbanking$secondaryArchSuffix = $portVersion
+	lib:libaqbanking$commandSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libgmp$secondaryArchSuffix
+	lib:libgwenhywfar$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libltdl$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libxmlsec1$secondaryArchSuffix
+	lib:libxslt$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	aqbanking${secondaryArchSuffix}_devel = $portVersion
+	cmd:aqbanking_config$commandSuffix
+	devel:libaqbanking$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	aqbanking$secondaryArchSuffix == $portVersion base
+	devel:libgmp$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	devel:libxmlsec1$secondaryArchSuffix
+	devel:libxslt$secondaryArchSuffix
+	devel:libgwenhywfar$secondaryArchSuffix
+	"
+
+SUMMARY_tools="Command line utilities for aqbanking"
+PROVIDES_tools="
+	aqbanking${secondaryArchSuffix}_tools = $portVersion
+	cmd:aqbanking_cli$commandSuffix
+	cmd:aqebics_tool$commandSuffix
+	cmd:aqhbci_tool4$commandSuffix
+	cmd:aqofxconnect_tool$commandSuffix
+	cmd:aqpaypal_tool$commandSuffix
+	"
+REQUIRES_tools="
+	haiku$secondaryArchSuffix
+	aqbanking$secondaryArchSuffix == $portVersion base
+	lib:libiconv$secondaryArchSuffix
```

Comment:
libiconv needs to be moved (sort alphabetically) :)
I'm out for a bit, so others can take over, otherwise will check up in the morning.

Reply:
Sorry, I saw your message about the alphabetical order when this was commited already, should have checked the guidelines first. It's done right in the last ones, I'll fix this soon.